src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-DCGMI.json

```diff
@@ -0,0 +1,37 @@
+﻿{
```

Comment:
Let's call this PERF-GPU-DCGMIDIAG, or QUAL-DCGMI-DIAG, if all it's doing is qualification

Reply:
I like QUAL-DCGMI-DIAG, I find it apt. Thanks for the suggestion Yang.